src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj

```diff
@@ -90,9 +89,11 @@
   </ItemGroup>
 
   <ImportGroup Condition=" '$(OS)' == 'Windows_NT' ">
-    <!--3 files are required for signing in this order: Signing.props, MicroBuild.Core.props, and MicroBuild.Core.targets-->
+    <!--5 files are required for signing in this order: Signing.props, MicroBuild.Core.props, and MicroBuild.Core.targets-->
     <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Signing.props'))\Signing.props" />
-    <Import Project="..\..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props" Condition="Exists('..\..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props')" />
-    <Import Project="..\..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.targets" Condition="Exists('..\..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.targets')" />
+    <Import Project="$(NuGetPackageRoot)MicroBuild.Core.0.3.0\build\MicroBuild.Core.props" Condition="Exists('$(NuGetPackageRoot)\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props')"/>
```

Comment:
Please try to remove all of these.
We should rely on implicit imports from the MicroBuild.Core package.